pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -134,6 +136,50 @@ message PeerInfo {
 }
 ```
 
+### Signature Policy
+
+The usage of the `signature`, `key`, `from`, and `seqno` fields in `Message` is now configurable.
+Initially this could be configured globally, however, configuration on a per-topic basis will facilitate mixed protocols better.
```

Comment:
```suggestion
Initially this could be configured globally, however, because the policies are mutually incompatible, configuration on a per-topic basis will facilitate mixed protocols better.
```